pkg/internal/testing/integration/internal/integration_tests/integration_test.go

```diff
@@ -19,6 +20,7 @@ import (
 
 var _ = Describe("The Testing Framework", func() {
 	var controlPlane *integration.ControlPlane
+	ctx := context.TODO()
```

Comment:
I opted to initially introduce contexts into test code in the most naive way possible, denoting it with `context.TODO()` so go linting tools can flag this as needing doing and it won't be forgotten.